src/main/resources/liquibase/crop_changelog/16_1_0.xml

```diff
@@ -0,0 +1,166 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="medramos" id="v16.1.0-1">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="germplsm" columnName="germplsm_uuid"/>
+			</not>
+		</preConditions>
+		<comment>
+			Add guid column to germplsm table
+		</comment>
+		<addColumn tableName="germplsm">
+			<column name="germplsm_uuid" type="VARCHAR(36)">
+				<constraints nullable="true"/>
+			</column>
+		</addColumn>
+	</changeSet>
+
+	<changeSet author="mderamos" id="v16.1.0-2">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT
+				COUNT(*)
+				FROM
+				information_schema.routines
+				WHERE
+				routine_schema = DATABASE()
+				AND specific_name = 'populate_germplsm_uuid_values';
+			</sqlCheck>
+		</preConditions>
+		<sql dbms="mysql" splitStatements="false">
+			DROP PROCEDURE populate_germplsm_uuid_values;
+		</sql>
+	</changeSet>
+
+	<changeSet author="mderamos" id="v16.1.0-3">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				SELECT
+				COUNT(*)
+				FROM
+				information_schema.routines
+				WHERE
+				routine_schema = DATABASE()
+				AND specific_name = 'populate_germplsm_uuid_values';
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Create Stored procedure that populate the germplsm_uuid values
+		</comment>
+		<sql dbms="mysql" splitStatements="false">
+			CREATE PROCEDURE populate_germplsm_uuid_values()
+			BEGIN
+
+			DECLARE useUuid INT DEFAULT 1;
+			DECLARE hasNullUuidCount INT DEFAULT 1;
+			DECLARE duplicates INT DEFAULT 1;
+
+			SELECT use_uuid
+			INTO useUuid
+			FROM workbench.workbench_crop
+			WHERE db_name = DATABASE();
+
+			IF useUuid = 0	THEN
+
+			WHILE(hasNullUuidCount > 0) DO
+
+			UPDATE germplsm g
+			SET germplsm_uuid = CONCAT((SELECT plot_code_prefix
+			FROM workbench.workbench_crop wc
+			WHERE db_name = DATABASE()),
+			'G',
+			RIGHT(sha1(CONCAT(uuid(), g.gid)), 15))
```

Comment:
In Java code, the suffix is based on RandomUtils.randomAlphanumeric not sha1. Is there any reason for that?

Reply:
It follows the same logic in generating the lotuuid. Since uuid() is a MySQL method, RandomUtils.randomAlphanumeric is used to somewhat replicate the generation of unique string using the said method.